react/features/base/react/components/web/NavigateSectionListItem.js

```diff
@@ -68,6 +85,7 @@ export default class NavigateSectionListItem extends Component<Props> {
                     className = 'navigate-section-tile-body'>
                     { duration }
```

Comment:
Do you know much about the data here and why an array is used with assumption about what each index in the array stands for? It looks like index 0 might be meeting url but the variable name is date. Index 1 might be data but the variable name is duration.